include/exadg/incompressible_navier_stokes/solver.h

```diff
@@ -87,6 +95,126 @@ run(std::string const & input_file,
     driver->print_performance_results(timer.wall_time());
 }
 
+void
+evaluate_convergence_study(SpatialResolutionParameters const &  spatial,
```

Comment:
I think this functionality should be realized as a separate PR. The present PR tends to be way too large.